### ISSUE TITLE
labels in the PixelCNN

### ISSUE THREAD

**Issue by ZhanYangen** [ritheshkumar95/pytorch-vqvae#16]
Hi, 
I have trained the VQVAE network on my own dataset comprise of 10,000 images of 64×64 pixels without any labels. In order to train PixelCNN network, I faked some labels like this:
`label_set=torch.zeros((10000,1), dtype=torch.int64)`
However, the shape of my faked labels seems not to fit in the code. In modules.py, there is this line `out_v = self.gate(h_vert + h[:, :, None, None])` in GatedMaskedConv2d.forward, where h is the label. In this way, the shape of h_vert would be (batch, 2×dim, 16, 16), but the shape of h would be (batch, 1, 2×dim).
So can anyone tell me how to deal with the labels?
Thanks.
